frontend/components/ui/navbar.tsx

```
@@ -0,0 +1,225 @@
+                                        <Link
+                                            href='#'
+                                            className='size-10 flex justify-center items-center border rounded text-gray-500 hover:bg-[#5751E1] hover:text-white'
+                                        >
+                                            <svg
+                                                xmlns='http://www.w3.org/2000/svg'
+                                                viewBox='0 0 24 24'
+                                                width='24'
+                                                height='24'
+                                                className='main-grid-item-icon size-5'
+                                                fill='currentColor'
+                                            >
```

Biome (1.9.4)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)